libs/ui/editor/src/lib/panel/teaserSelectPanel.tsx

```
@@ -657,6 +692,33 @@ export function TeaserSelectPanel({onClose, onSelect}: TeaserSelectPanelProps) {
+          <>
+            {zones.map(({name, zoneId, width, height}) => {
+              return (
+                <AdTeaserWrapper
+                  style={{aspectRatio: maxWidth / maxHeight}}
+                  onClick={() =>
+                    onFinalSelect({
+                      type: TeaserType.Advertisement,
+                      zoneId,
+                      style: TeaserStyle.Default
+                    })
+                  }>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)